### PR TITLE
Document the preloading metadata within SCSS

### LINKED ISSUE #132: 5.4: Document the preloading support
https://github.com/WoltLab/WCF/pull/4029

### PR DESCRIPTION
**Pull request by TimWolla**
Resolves #132

### CODE REVIEW

File: docs/view/css.md [hunk 1]
```diff
@@ -53,3 +53,31 @@ Media breakpoints instruct the browser to apply different CSS depending on the v
 | `screen-md-down` | Smartphones and Tablets | `(max-width: 1024px)` |
 | `screen-md-up` | Tablets (landscape) and desktop PC | `(min-width: 769px)` |
 | `screen-lg` | Desktop PC | `(min-width: 1025px)` |
+
+## Asset Preloading
+
+WoltLab Suite’s SCSS compiler supports adding [preloading](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content) metadata to the CSS.
+To communicate the preloading intent to the compiler the `--woltlab-suite-preload` CSS variable is set to the result of the `preload()` function:
```
Comment (ghost) [inline docs/view/css.md]:
```suggestion
To communicate the preloading intent to the compiler, the `--woltlab-suite-preload` CSS variable is set to the result of the `preload()` function:
```

File: docs/view/css.md [hunk 2]
```diff
@@ -53,3 +53,31 @@ Media breakpoints instruct the browser to apply different CSS depending on the v
...
+
+```scss
+body {
```
Comment (ghost) [inline docs/view/css.md]: I would use a different selector (`.fooBar`) to emphasize that this is an example and no existing code.

**Review by ghost — CHANGES_REQUESTED** [review]
Everything else LGTM

File: docs/view/css.md (hunk 2 above)
Comment (TimWolla) [inline docs/view/css.md]:
```suggestion
.fooBar {
```

**Pull request merged**